src/Range.ts

```diff
@@ -5,46 +5,36 @@ export class Range {
      * @param _limits inclusive limits. All numbers outside of it are nor in nor out of the range
      * @param _range range inside of the limits. All ranges are inclusive.
      */
-    constructor(private _limits: [number, number], private _range: Array<[number, number]>) {
+    constructor(private _limits: [number, number], private _range: Array<[number, number]>, private _inverted = false) {
         // blank
     }
 
     in(val: number): boolean {
-        let index = _.sortedIndexBy(this._range, [val, val], item => item[1]);
-        if (index >= this._range.length) {
-            index = this._range.length - 1;
+        if (!this._checkLimits(val)) {
+            return false;
+        } else {
+            let result = false;
+            let index = _.sortedIndexBy(this._range, [val, val], item => item[1]);
+            if (index === this._range.length) {
+                result = false;
+            } else {
+                const range = this._range[index];
+                result = _.inRange(val, range[0], range[1] + 1);
```

Comment:
@dryganets I think you do not need to _import_ entire `lodash`. 

```
import sortedIndexBy = require('lodash/sortedIndexBy');
import inRange = require('lodash/inRange');
```

Does that make sense?

Reply:
I had a lodashMini file one iteration ago.
@berickson1 told me that this is fixed by custom babel plugin and we don't need to use this pattern anymore

Reply:
@berickson1 @dryganets you are right, users can use plugins like `babel-lodash-plugin`. `regexp-i18n` does not use any additional build tools - only ts comiler., I've made a small investigation with two versions of `regexp-i18n`:
 
1. where there is full `lodash` import
2. where `regexp-i18n` uses only necessary functions - https://github.com/a-tarasyuk/regexp-i18n/tree/feature/lodash-cherry-pick

both versions were tested with `webpack + babel`, results you can find here - https://github.com/a-tarasyuk/bundle-size-test. Based on results, seems, we can decrease bundle size by half., what do you think about this case? Thanks in advance.